app/pkg/traceroute/gotraceroute.go

```diff
@@ -21,13 +21,6 @@ func printHop(hop Hop) {
 }
 
 func GoTraceroute(host string, options Options, pathToSave string) {
```

Comment:
do not ignore error ;) especially when you deal with reading/writing files or json Marshal/Unmarshal :D